app/settings.py

```diff
@@ -26,6 +26,8 @@
 STAGING = os.environ['STAGING']
 
 COLLISION_MAX_RETRY = 10
-SHORT_ID_SIZE = 10
+
+SHORT_ID_SIZE = os.getenv('SHORT_ID_SIZE', '12')
```

Comment:
I would do the conversion to `int` and not in the code.

Reply:
Thanks. Already did in the meantime